src/i18n/Messages/Symfony/FlushInvalidatedResource.php

```diff
@@ -71,6 +73,7 @@ public function serialize()
      */
     public function unserialize($serialized)
     {
+        Deprecation::notice('4.12.0', 'Will be removed in 5.0');
```

Comment:
Add version to the phpdoc and update the message in both to be standardised

Reply:
Updated